newrelic/hooks/framework_django.py

```diff
@@ -458,8 +463,14 @@ def instrument_django_core_handlers_base(module):
     wrap_post_function(module, 'BaseHandler.load_middleware',
             insert_and_wrap_middleware)
 
-    wrap_function_wrapper(module, 'BaseHandler.get_response',
-            _nr_wrapper_BaseHandler_get_response_)
+    if six.PY3 and hasattr(module.BaseHandler, '_get_response') and hasattr(module.BaseHandler, '_get_response_async'):
+        wrap_function_wrapper(module, 'BaseHandler._get_response',
+                _nr_wrapper_BaseHandler_get_response_)
+        wrap_function_wrapper(module, 'BaseHandler._get_response_async',
+                _nr_wrapper_BaseHandler_get_response_async_)
+    else:
+        wrap_function_wrapper(module, 'BaseHandler.get_response',
+                _nr_wrapper_BaseHandler_get_response_)
```

Comment:
This logic is actually incorrect so I'm glad we took a second look at this. I believe the correct thing to do is to be wrapping `BaseHandler.get_response` and `BaseHandler.get_response_async` because the intent is to have an instrumentation point that is used just before the request is complete so that we can ignore errors based on status code.